Library/Formula/ola.rb

```diff
@@ -5,6 +5,7 @@ class Ola < Formula
   sha256 "621f18f591a418236595d0117b4ab16d8e39a69b03071e62fdae0e9b01533de0"
 
   bottle do
+    revision 1
```

Comment:
OK, I put this line back. This seems to be a change in procedure because on a previous update of this formula, I was asked to manually remove the revision tag when I submitted a change which updated the JAR version and there was one present.